src/main/java/net/openhft/chronicle/core/util/WeakReferenceCleaner.java

```diff
@@ -100,7 +105,7 @@ public void run() {
                     }
 
                 } catch (InterruptedException e) {
-                    LOGGER.warn("Interrupted while trying to retrieve reference, exiting.", e);
+                    LOGGER.debug("Interrupted while trying to retrieve reference, exiting.", e);
```

Comment:
This is an expected condition when the JVM shuts down, right? Hence moving to debug.